### PR TITLE
Leaderboard: Final touches

### PR DESCRIPTION
**Pull request by fasaldana**
## Changed basic style for the page:
* Background color of the page.
* Animation when buttons are clicked.
* Color of the headers for the table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,12 +36,6 @@ For testing you can do it with the command:
 
 - npm start
```
Comment (medaminedev66) [inline README.md:38]: - [ ] This command is for running the app not the test. Please change `Run tests` to `Usage` and add another step: `npm install`

File: README.md [hunk 2]
```diff
@@ -54,7 +48,7 @@ You can build the project using the command:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/fasaldana/Leaderboard/issues).
 
 ## Show your support
```
Comment (medaminedev66) [inline README.md:55]: - [ ] The `MIT` link is broken, Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @fasaldana  😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Meltrust** [comment]
Hello @fasaldana, I can see that you deleted the link to the license instead of fixing it.  If you don't know how to add a license to your project, please contact me on Slack, I will help you!

I can wait for 10 minutes, otherwise, I will be forced to request changes, in accordance with the last reviewer's request.

I already contacted you, please respond.

**Review by Meltrust — APPROVED** [review]
Hi @fasaldana,


## Wow, you did it 🎉
Brilliant
## Thank you for improving and correcting your readme file, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**